internal/pkg/template/templates/workloads/partials/cf/rollback-alarms.yml

```diff
@@ -0,0 +1,45 @@
+{{- if .DeploymentConfiguration.CPUUtilization}}
+CPURollbackAlarm:
+  Metadata:
+    'aws:copilot:description': "A CloudWatch alarm associated with CPU utilization for deployment rollbacks"
+  Type: AWS::CloudWatch::Alarm
+  DependsOn: Service
+  Properties:
+    AlarmDescription: "Roll back ECS service if CPU utilization is greater or equal to {{.DeploymentConfiguration.CPUUtilization}}% twice in 90 seconds."
```

Comment:
Is 90 secs default value given by the deployments team?

Reply:
No, the deployments team has the bake time adjusting according to alarms. We need to make the evaluation time long enough to get proper data, but not so long that deployments always take forever. It's hard to know how to hit the balance. ECS emits data to CloudWatch each minute, so I changed this from 90 seconds.